vendor/engines/secure_rooms/app/presenters/secure_rooms/link_collection_extension.rb

```diff
@@ -0,0 +1,23 @@
+module SecureRooms
+
+  module LinkCollectionExtension
+
+    extend ActiveSupport::Concern
+
+    included do
+      insert_index = tab_methods.index(:admin_products) || -1
+      tab_methods.insert(insert_index, :admin_occupancies)
+    end
+
+    def admin_occupancies
+      if single_facility? && SecureRoom.for_facility(facility).exists?
```

Comment:
Can we also add a `not_archived` scope on here? I think if you deactivate a room, we should let this go away. I wouldn't use `Product.active` because it also excludes hidden, but not-archived instruments. There's definitely some awkwardness to the names there.